src/components/admin/article/articleLogic.ts

```diff
@@ -127,12 +197,13 @@ export function buildJsonGraphEnvelope(componentState, articleId, markdown) {
   jsonGraphEnvelope.jsonGraph.articles.byId[articleId].markdown = markdown;
   jsonGraphEnvelope.jsonGraph.articles.byId[
     articleId
-  ].html = this.converter.makeHtml(markdown);
```

Comment:
We missed this in my original PR @Helw150, of course no `this` here, but JS is super weak and let's you do all kinds of wrong things :'( 

So yay Typescript which will tell us all of this from now on! :D